package.json

```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "@next/third-parties": "^15.2.0",
+    "antd": "^5.25.0",
```

Comment:
@ansivgit @Quiddlee Are you guys okay with using an external component library for pagination?

Reply:
Actually, I'm not sure that we need it